### PR TITLE
[biomed nl] update embedding index with linked props examples

### PR DESCRIPTION
**Pull request by chejennifer**

### CODE REVIEW

File: tools/nl/embeddings/data/curated_input/bio/sheets_svs.csv [hunk 1]
```diff
@@ -1,45 +1,48 @@
...
+geneID,geneID,gene id,,
+ncbiProteinAccessionNumber,ncbiProteinAccessionNumber,NCBI protein accession number,,
+alleleOrigin,alleleOrigin,Variant allele origin,,Origin of variant allele
+alleleType,alleleType,The allele of a genetic variant observed within a population,,Type of allele
+ncbiDNASequenceName,ncbiDNASequenceName,NCBI defined segment of DNA sequence name,,Name used by NIH NCBI to refer to a segment of DNA sequence
+imageUrl,imageUrl,url to an image of what the biological specimen looks like,,what the entity looks like
+genomicCoordinates,genomicCoordinates,genomic coordinates,,
+availableStrength,availableStrength,dose approved for a drug,,
+<-referenceSNPClusterID{typeOf:GeneticVariantGeneAssociation}->geneSymbol,GeneticVariantGeneAssociation,Gene associated with a genetic variant,,
+<-geneSymbol{typeOf:GeneticVariantGeneAssociation}->referenceSNPClusterID,GeneticVariantGeneAssociation,genetic variant associated with a gene,,
```
Comment (pradh) [inline tools/nl/embeddings/data/curated_input/bio/sheets_svs.csv:45]: I would recommend dropping a README in this folder to explain these links?

**Review by pradh — APPROVED** [review]
Nice!

File: tools/nl/embeddings/data/curated_input/bio/sheets_svs.csv (hunk 1 above)
Comment (chejennifer) [reply tools/nl/embeddings/data/curated_input/bio/sheets_svs.csv:45]: done

File: tools/nl/embeddings/data/curated_input/bio/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Curated Input for Bio index
+
+This index has properties used by biomedical entities and follows the format of [relation expressions](https://docs.datacommons.org/api/rest/v2#relation-expressions). Properties can be structured like:
+
+- `prop`: this can match to either in or out properties
+- `->prop`: this matches to an 'out' property
+- `<-prop`: this matches to an 'in' property
+- `<-prop1{typeOf:X}->prop2`: in this case, we will get all the 'in' values for prop1 that are of type X & then from those values, get all the 'out' values for prop2
```
Comment (pradh) [inline tools/nl/embeddings/data/curated_input/bio/README.md:8]: Provide a couple of examples?

**Review by pradh — APPROVED** [review]

File: tools/nl/embeddings/data/curated_input/bio/README.md (hunk 2 above)
Comment (chejennifer) [reply tools/nl/embeddings/data/curated_input/bio/README.md]: done

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**